tests/topotests/lib/topojson.py

```diff
@@ -151,24 +144,24 @@ def link_sort(x):
                     destRouter = destRouterLink
                     curRouterLink = curRouter
 
+
                 if destRouter in listRouters:
-                    currRouter_link_json = topo["routers"][curRouter]["links"][
-                        destRouterLink
-                    ]
-                    destRouter_link_json = topo["routers"][destRouter]["links"][
-                        curRouterLink
-                    ]
+
```

Comment:
Whitespace changes are fine, but need to be *pulled* out into their own commits.

Reply:
Removed whitespace from this commit. We will take care in another commit.